src/main/java/org/folio/rest/impl/ModTenantAPI.java

```diff
@@ -118,8 +119,8 @@ private void postConfig(ConfigurationsClient client, String configName, Promise<
     try {
       client.postConfigurationsEntries(null, config, resp -> {
         if (resp.statusCode() != 201) {
-          logger.error("Invalid response from mod-configuration. Cannot post config '{}': {} {}", configName, resp.statusCode(),
-              resp.statusMessage());
+          logger.error(String.format("Invalid response from mod-configuration. Cannot post config '%s'." +
```

Comment:
Probably better to stick with slf4j concatenation... see https://stackoverflow.com/questions/10555409/logger-slf4j-advantages-of-formatting-with-instead-of-string-concatenation

Reply:
It was not working, that's why I changed it. I'll double check.